cmd/oras/internal/option/remote.go

```diff
@@ -62,6 +64,7 @@ func (opts *Remote) EnableDistributionSpecFlag() {
 func (opts *Remote) ApplyFlags(fs *pflag.FlagSet) {
 	opts.ApplyFlagsWithPrefix(fs, "", "")
 	fs.BoolVarP(&opts.PasswordFromStdin, "password-stdin", "", false, "read password or identity token from stdin")
+	fs.StringArrayVarP(&opts.headerFlags, "header", "H", []string{}, "add custom headers to requests")
```

Comment:
nit: in golang, nil slice is still a valid slice.
```suggestion
	fs.StringArrayVarP(&opts.headerFlags, "header", "H", nil, "add custom headers to requests")
```